<ui version="4.0" >
 <class>SimpleMCModelerWidget</class>
 <widget class="QWidget" name="SimpleMCModelerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Number of discretization steps</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="sliderDSteps" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QSpinBox" name="numdstepsEdit" >
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Number of states for discretization</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSlider" name="sliderStates" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QSpinBox" name="numstatesEdit" >
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>61</width>
       <height>145</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2" >
    <widget class="QPushButton" name="computeButton" >
     <property name="text" >
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="progressLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderDSteps</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numdstepsEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>253</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel" >
     <x>419</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numdstepsEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderDSteps</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>413</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel" >
     <x>285</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderStates</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numstatesEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>222</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>435</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numstatesEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderStates</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>439</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel" >
     <x>291</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
